Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -0,0 +1,213 @@
+1. You can make sure that the hologram is only reacting to the user's eye gaze.
+2. Several Unity events have already been set up to make it fast and convenient to handle and reuse existing behaviors.
+
+#### Example: Attentive Notifications
+For example, in [mrtk_eyes_02_TargetSelection.unity](/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Scenes/mrtk_eyes_02_TargetSelection.unity), 
+you can find an example for _'smart attentive notifications'_ that react to your eye gaze. 
+These are 3D text boxes that can be placed in the scene and that will smoothly enlarge and turn toward the user when being looked at to ease legibility.
+While the user is reading the notification, the information keeps getting displayed crisp and clear. 
+After reading it and looking away from the notification, the notification will automatically be dismissed and fades out.
+To achieve all this, we created a few generic behavior scripts that are not specific to Eye Tracking at all such as:
+- [FaceUser.cs](/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/FaceUser.cs)
+- [ChangeSize.cs](/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/ChangeSize.cs)
+- [BlendOut.cs](/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/BlendOut.cs)
+
+The advantage of this approach is that the same scripts can be reused by various events. 
+For example, a hologram may start facing the user based on a voice commands or after pressing virtual button. 
+To trigger these events, you can simply reference the methods that should be executed in the [EyeTrackingTarget](/Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/EyeTrackingTarget.cs) 
+script that is attached to your GameObject.
+For the example of the _'smart attentive notifications'_, the following happens:
+- **OnLookAtStart()**: The notification starts to...
+    - *FaceUser.Engage:* ... turn toward the user.
+    - *ChangeSize.Engage:* ... increase in size _(up to a specified maximum scale)_.
+    - *BlendOut.Engage:* ... starts to blend in more _(after being at a more subtle idle state)_.  
+
+- **OnDwell()**: Informs the _BlendOut_ script that the notification has been looked at sufficiently.
+
+- **OnLookAway()**: The notification starts to...
+    - *FaceUser.Disengage:* ... turn back to its original orientation.
+    - *ChangeSize.Disengage:* ... decrease back to its original size.
+    - *BlendOut.Disengage:* ... starts to blend out - If _OnDwell()_ was triggered, blend out completely and destroy, otherwise back to its idle state.
+
+**Design Consideration:**
+The key to an enjoyable experience here is to carefully tune the speed of any of these behaviors to avoid causing discomfort by reacting to the user’s eye gaze too quickly all the time. 
+Otherwise this can quickly feel extremely overwhelming.
+
+![MRTK](/External/ReadMeImages/EyeTracking/mrtk_et_EyeTrackingTarget_Notification.jpg =750x450)
```

Comment:
This is showing up on the preview as text, not an image. Maybe the link is broken?

Reply:
Should be fixed!